store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -77,6 +85,24 @@ public void testWriteAndReadFiles() throws IOException, InterruptedException {
     Assert.assertEquals(i, 100);
 
     reader.close();
+
```

Comment:
This test case points to sequential read. One reader gets closed and second one starts. What exactly happens when there is parallel read of two readers. Can we have a test case for that?

Reply:
Ok, I will add. What's more, search mode has used CarbonRecordReader, there are some test case to concurrent run in org.apache.carbondata.examples.SearchModeExample.